examples/reservation.py

```diff
@@ -14,11 +14,18 @@
 from braket.aws import AwsDevice
 from braket.circuits import Circuit
 from braket.devices import Devices
+from braket.reservations import DirectReservation
 
 bell = Circuit().h(0).cnot(0, 1)
 device = AwsDevice(Devices.IonQ.Aria1)
 
 # To run a task in a device reservation, change the device to the one you reserved
-# and fill in your reservation ARN
+# and fill in your reservation ARN.
+with DirectReservation(device, reservation_arn="reservation ARN"):
+    task = device.run(bell, shots=100)
+print(task.result().measurement_counts)
+
+# Alternatively, you may start the reservation globally
+DirectReservation(device, reservation_arn="reservation ARN").start()
 task = device.run(bell, shots=100, reservation_arn="reservation ARN")
```

Comment:
Reservation ARN technically isn't needed for this use case, but we should still have an example for creating tasks with a reservation ARN

Reply:
good catch! will fix this